Makefile

```diff
@@ -158,7 +158,7 @@ WEBHOOK_ROOT ?= $(MANIFEST_ROOT)/webhook
 RBAC_ROOT ?= $(MANIFEST_ROOT)/rbac
 ASO_CRDS_PATH := $(MANIFEST_ROOT)/aso/crds.yaml
 ASO_VERSION := v2.5.0
-ASO_CRDS := resourcegroups.resources.azure.com natgateways.network.azure.com managedclusters.containerservice.azure.com managedclustersagentpools.containerservice.azure.com bastionhosts.network.azure.com privateendpoints.network.azure.com
+ASO_CRDS := resourcegroups.resources.azure.com natgateways.network.azure.com managedclusters.containerservice.azure.com managedclustersagentpools.containerservice.azure.com bastionhosts.network.azure.com privateendpoints.network.azure.com fleets.containerservice.azure.com fleetsmembers.containerservice.azure.com
```

Comment:
Does CAPZ create `fleets` or just `fleetsmembers`? If we don't need `fleets` I'd be +1 to removing that just to keep the default CRD set as minimal as possible.

Reply:
CAPZ only creates fleetsmbers but I have to add fleets to the CRDs since fleetsmembers depend on them

Reply:
In what way does the fleetsmembers CRD depend on fleets? Does ASO throw an error when it starts up?

Reply:
Yeah, ASO gives me an error when I try to create a `fleetsmember`. It has been a while since I made this change but I can try and reproduce it and get you the exact error.

Reply:
Nevermind it looks like it works without it being defined.. I'm not sure what happened earlier. Good catch!